webui/src/lib/components/repository/tree.jsx

```diff
@@ -25,7 +25,7 @@ import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Dropdown from "react-bootstrap/Dropdown";
 
-import { commits, linkToPath } from "../../api";
+import {config, commits, linkToPath, objects} from "../../api";
```

Comment:
do you know what the linter shows this message= `'config' is defined but never used`?

Reply:
ha ok, got it, it's not the same config
please remove if not needed 🙂